nixos/modules/services/networking/firefox/sync-server.nix

```diff
@@ -70,18 +72,6 @@ in
         '';
       };
 
-      user = mkOption {
-        type = types.str;
-        default = "syncserver";
-        description = "User account under which syncserver runs.";
-      };
-
-      group = mkOption {
-        type = types.str;
-        default = "syncserver";
-        description = "Group account under which syncserver runs.";
-      };
```

Comment:
Can you add this to `nixos/modules/rename.nix` using `mkRemovedOptionModule`?

Reply:
Done :)